src/components/Header.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <div class="signed-header flex bg-white shadow-lg">
+    <nav class="container m-auto flex justify-between items-center">
+      <div class="logo" />
+      <div>
+        settings
+      </div>
+    </nav>
+  </div>
+</template>
+
+<script>
+
+export default {
+  name: "Header"
```

Comment:
Vue recommends having a prefix to any component. Call it `AppHeader` or `FbHeader` (FB - Facebook)
Same for footers, buttons, etc... `FbButton`